qa/tasks/ceph_manager.py

```diff
@@ -239,6 +239,22 @@ def run_ceph_objectstore_tool(self, remote, osd, cmd):
                 stdout=StringIO(),
                 stderr=StringIO())
 
+    def run_ceph_bluestore_tool(self, remote, osd, cmd):
+        if self.ceph_manager.cephadm:
+            return shell(
+                self.ceph_manager.ctx, self.ceph_manager.cluster, remote,
+                args=['ceph-bluestore-tool', '--err-to-stderr'] + cmd,
+                name=osd,
+                wait=True, check_status=False,
```

Comment:
instead of raising a separate exception in several of the callers, you could add an argument for check_status so we get the usual CommandFailedError with the full command run and appropriate grouping in sentry